src/common/misc.c

```diff
@@ -664,6 +664,8 @@ daos_prop_free(daos_prop_t *prop)
 		switch (entry->dpe_type) {
 		case DAOS_PROP_PO_LABEL:
 		case DAOS_PROP_CO_LABEL:
+		case DAOS_PROP_PO_OWNER:
+		case DAOS_PROP_PO_OWNER_GROUP:
```

Comment:
just curious why the OWNER, GROUP stored by a string rather than uid/gid?

Reply:
It has to do with our future goals for access control. Right now it's just Unix users/groups, but we'd like to support other authentication mechanisms in the future (Kerberos has come up as an example) which may not map directly to one specific uid/gid.